Makefile

```diff
@@ -51,7 +52,7 @@ setup-env:
 
 .PHONY: .test-ci
 .test-ci:
-	go test -count=1 -cover ./...
+	go test -v -cover ./...
```

Comment:
-count=1 causes this to run even when it would otherwise assume that this test is cached. I guess there might be a reason for this

Reply:
It isn't needed in CI and it was a leftover from my last PR, for example, it wasn't there a last week.

Reply:
:+1: